supervisor/addons/addon.py

```diff
@@ -417,7 +414,7 @@ async def install_apparmor(self) -> None:
         exists_addon = self.path_apparmor.exists()
 
         # Nothing to do
-        if not exists_local and not exists_addon:
+        if not (exists_local or exists_addon):
```

Comment:
Without the parenthesis the expression can be easily read from left to right in one pass. That's not possible with the parenthesis.